src/client/VZSidebar/styles.scss

```diff
@@ -1,4 +1,56 @@
 .vz-sidebar {
+  .list {
+    list-style: none;
+  }
+  
+  .list li {
+    outline: none;
+    cursor: default;
+  }
+  
+  .list li:nth-child(2n) {
+    background: var(--spectrum-alias-highlight-hover);
```

Comment:
Where is the `--spectrum-alias-highlight-hover` definition coming from? Did you verify that this resolves correctly? We may need a CSS import to make this work.